Packager/ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1 @@
-m_EditorVersion: 5.2.0f3
```

Comment:
I think we want to keep the Packager at the oldest version that NuGetForUnity intends to support.